forem/forem
# GDPR Inflections

## What type of PR is this? (check all applicable)


## Description
It came up in [this PR](https://github.com/forem/forem/pull/17538#discussion_r864598236) that we're titlizing the sidebar navigation items, and hence "GDPR delete requests".titleize will result in "Gdpr Delete Requests". 

Hence, we want to create an inflection for the GDPR Delete Requests and amend the class names in the files as per this rule. 

## Related Tickets & Documents



- Related Issue #
- Closes #

## QA Instructions, Screenshots, Recordings

1. Start up the app, and navigate to the [GDPR Delete Requests page](http://localhost:3000/admin/users/gdpr_delete_requests). Click around and make sure that everything still works. 

2. In addition, I changed a migration file so run `rake:db:drop; rake db:create; rake db:migrate; rake db:seed` and make sure everything works. 


### UI accessibility concerns?

None

## Added/updated tests?

      have not been included_

## [Forem core team only] How will this change be communicated?

_Will this PR introduce a change that impacts Forem members or creators, the
development process, or any of our internal teams? If so, please note how you
will share this change with the people who need to know about it._

      [Storybook](https://storybook.forem.com/) (for Crayons components)
      updated. I have filled out the
      [Changes Requested](https://github.com/forem/admin-docs/issues/new?assignees=&labels=&template=changes_requested.md)
      issue template so Community Success can help update the Admin Docs
      appropriately.
      [`CHANGELOG.md`](https://github.com/forem/forem/tree/main/CHANGELOG.md)
      or in a [forem.dev](http://forem.dev) post
      replace this line with details on why this change doesn't need to be
      shared_

## [optional] Are there any post deployment tasks we need to perform?

## [optional] What gif best describes this PR or how it makes you feel?

![alt_text](https://media.giphy.com/media/coijcQneuNxPTLWPoQ/giphy.gif)
